docs/index.md

```diff
@@ -384,6 +384,12 @@ Spine Default Animation
 Skin
 : The skin to use for the animation when the scene is initialized.
 
+Create Bones
+: When enabled, creates individual GUI nodes for each Spine bone under the Spine node. This allows parenting other GUI nodes to bones and using functions such as [`gui.get_spine_bone()`](/extension-spine/gui_api#gui.get_spine_bone).
+
+- Default for new nodes: Off. Keeps node count low and improves performance.
+- Note: If disabled, functions that require per-bone GUI nodes (e.g. `gui.get_spine_bone`, addressing nodes as `spine_node_id/bone_name`) will not work.
+
```

Comment:
> Default for new nodes: Off

Does this mean that the default for existing nodes is supposed to be On? Because the field defaults to `false` in `gui_ddf.proto`, which would make the default `false` for existing content too.

Reply:
We did it like that for spine components. It is a breaking change and we are fine with it because in most of the cases  you don't need bones. We will release it with "breaking change" label for the extension